src/ResourceManager/Sql/Commands.Sql/Commands.Sql.csproj

```diff
@@ -210,6 +210,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.Azure.Common.2.1.0\lib\net45\Microsoft.Azure.Common.NetFramework.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Azure.Management.Storage">
```

Comment:
@yoavrubin You will need to update the installer wix file to install this new dll.  Please rebuild and recreate the installer wix suing tools\BuildInstaller.ps1